pytrip/cube.py

```diff
@@ -745,8 +746,11 @@ def _write_trip_header(self, path):
         else:
             output_str += "z_table no\n"
 
-        with open(path, "w+", newline='\n') as f:
-            f.write(output_str)
+        with io.open(path, "w+", newline='\n') as f:
```

Comment:
```suggestion
        # for compatibility with python 2.7 we need to use `io.open` instead of `open`, as `open` function in python 2.7 cannot handle `newline` argument. This needs to be followed by `decode()`d string being written
        with io.open(path, "w+", newline='\n') as f:
```